client/src/utils/attributeAndParamUtils.js

```diff
@@ -256,7 +256,7 @@ const getParameterObject = (robotId, activityId) => {
  * @param {String} activityId Id of the activity for which to change the value for
  * @param {Object} value The value object returned by the dropdown selection
  */
-const setSingleParameter = (activityId, value) => {
+const setSingleParameter = (activityId, value, paramName) => {
```

Comment:
I do often do it as well but it is better if write parameterName instead of paramName. Please check all the occurences of paramName and change it to parameterName

Reply:
Good point! I Just renamed it! :)